migrations/2024-10-30-155932_increase_post_url_max_length/down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE post
```

Comment:
Renamed the folder so that ci can test the migration. Have to rename it back once it passes.

Reply:
Just want to note, that I manually ran changing the column back to `varchar(512)` in prod, and things *still* stayed slow. 

So even if we do this manually on the DB, we need to take a DB backup before deploying this again.